reactivated/apps.py

```diff
@@ -120,11 +125,20 @@ def get_interfaces() -> Dict[str, Tuple[Any]]:
 
 
 def get_values() -> Dict[str, Any]:
-    return value_registry
+    serialized = {}
```

Comment:
Maybe export could do the check whether something is serializable or not, then we just call export inside the RPC registrations.